include/llarp/dns.hpp

```diff
@@ -84,12 +86,24 @@ struct dns_packet
   std::vector< std::unique_ptr< dns_msg_answer > > additional_rrs;
 };
 
+std::vector< byte_t >
+packet2bytes(dns_packet &in);
+
 std::string
 getDNSstring(const char *const buffer, uint32_t *pos);
 
 void
 code_domain(char *&buffer, const std::string &domain) throw();
 
+void
+vcode_domain(std::vector< byte_t > &bytes, const std::string &domain) throw();
```

Comment:
`throw()` specifiers are deprecated in later versions of the standard, since the introduction of `noexcept`.

Reply:
so should I remove throw()? or replace it with noexcept? Again either Jeff code or was just copying patterns from Jeff.